docs/reference/contributing/guidelines/workflow.md

```diff
@@ -106,12 +106,14 @@ Every pull request changing or adding functionality must contain a release notes
 
 It must contain:
 
-- A brief description of changes introduced.
-- An analysis of effects: components affected, potential consequences for users and reasons for the addition or change.
+- A brief description of changes introduced, including justification description.
+- An analysis of effects: components affected and potential consequences for users.
 - Migration guidance: actions for updating the current code. Please include code snippets to illustrate before and after the addition or change.
 
 <span class="notes">**Note:** We may use this content in our official release notes.</span>
 
+For more details, please see the [pull request template addition for functional changes](./pull_request_drafts/pr_functional_change.md).
```

Comment:
This link 404s. When I checked, it's because the file it's pointing to - the PR draft - isn't in the JSON. Was there a reason for that?

Reply:
@0xc0170 Do you know why this link is failing ?

Reply:
The link is failing because the file isn't part of the docs build. 
My question is: why isn't it part of the docs build? Is that on purpose?

Reply:
How to fix it? adding it to the docs.json file - I was not aware of it so missed that part

Reply:
I'll fix it; just wanted confirmation it needs to be there and wasn't being held back for a specific reason.

Reply:
Would this fix it: https://github.com/ARMmbed/mbed-os-5-docs/compare/development...0xc0170:fix_workflow_drafts?expand=1 ?

Reply:
Yes (I'll fix the link itself once that's merged)

Reply:
OK, please fix, thank you!